src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -50,12 +75,28 @@ String profileOwnerName = (String) request.getAttribute("profileOwner");
         <button type="submit">Submit</button>
       </form>
     <% } %>
-  </div>
+    <h2><%= profileOwnerName %>'s sent messages</h2>
 
-  <div id="container">
-    <h1>Messages will go here?</h1>
+    <!--This is where the user's sent messages will show up -->
+    <div id="messages">
+      <ul>
+    <% //This queue acts as a priority queue. Everytime you pop, the oldest message is returned
+       Queue<Message> user_messages = MessageStore.getInstance()
```

Comment:
Camelcase instead of snakecase for variable naming in JSP files, see: http://www.oracle.com/technetwork/articles/java/code-convention-138726.html